packages/components/src/internal/test-utils.tsx

```diff
@@ -2,7 +2,6 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
should this be commented out?

Reply:
Yes. This is an [ESLint configuration comment](https://eslint.org/docs/latest/use/configure/rules#using-configuration-comments) used to override an ESLint rule for this file.